src/Events/CartEmptied.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace YiddisheKop\LaravelCommerce\Events;
+
+use Illuminate\Queue\SerializesModels;
+use YiddisheKop\LaravelCommerce\Models\Order;
+use Illuminate\Foundation\Events\Dispatchable;
+
+class CartEmptied
+{
+    use Dispatchable;
+    use SerializesModels;
+
+    public function __construct(
+        public Order $order,
```

Comment:
This package currently supports php 7.4

If we use php 8 features, we'd need to remove php 7.4 support from `composer.json`.

It's a breaking change, so we'd need to tag a new version.